releasenotes/notes/fix-mpl-draw-digraph-plots-aecf86738ab9b0db.yaml

```diff
@@ -0,0 +1,10 @@
+---
+fixes:
+  - |
+    Fixes the plots of multigraphs using the `mpl_draw()` function. Refer to
+    `#12345 <https://github.com/Qiskit/rustworkx/issues/774>` for more
+    details.
```

Comment:
The release notes would be better if you actually drew a multigraph with `mpl_draw` as an example. You can use https://github.com/Qiskit/rustworkx/blob/53545127b2235b8f5eb517fefe0502029fdb69d7/releasenotes/notes/0.14/add-albert-graph-5a7d393e1fe18e9d.yaml#L16 as an example, it becomes this: https://www.rustworkx.org/release_notes.html#new-features